resources/provider/moduledata/drift/2/drift.hcl

```diff
@@ -0,0 +1,1913 @@
+    filters = [
+      "c.global_table_version='2017.11.29'"
+    ]
+
+    iac {
+      terraform {
+        type = "aws_dynamodb_global_table"
+        identifiers = [ "replication_instance_arn" ]
+      }
+    }
+  }
+
+  # TODO: ec2.byoip_cidrs (no data in tests)
+
+  resource "ec2.customer_gateways" {
+    iac {
+      terraform {
+        type = "aws_customer_gateway"
+      }
+    }
+  }
+
+  resource "ec2.ebs_snapshots" {
+    ignore_attributes = [ "start_time" ]
+
+    iac {
+      terraform {
+        type = "aws_ebs_snapshot"
+      }
+    }
+  }
+
+  resource "ec2.ebs_volumes" {
+    iac {
+      terraform {
+        type = "aws_ebs_volume"
+      }
+    }
+  }
+
+  resource "aws_ec2_ebs_volume_attachments" {
+    identifiers = [ "instance_id", "volume_id", "device" ]
+    iac {
+      terraform {
+        type = "aws_instance"
+        path = "root_block_device"
+        identifiers =  [ "root.id", "volume_id", "device_name" ]
+      }
+    }
+  }
+
+  resource "ec2.eips" {
+    ignore_attributes = [ "network_interface_owner_id" ]
+    iac {
+      terraform {
+        type = "aws_eip"
+        identifiers = [ "allocation_id" ]
+        attribute_map = [
+          "network_interface_id=network_interface",
+          "private_ip_address=private_ip"
+        ]
+      }
+    }
+  }
+
+  resource "ec2.flow_logs" {
+    iac {
+      terraform {
+        type = "aws_flow_log"
+      }
+    }
+  }
+
+  resource "ec2.images" {
+    identifiers = [ "id", "region" ]
+    iac {
+      terraform {
+        type = "aws_imagebuilder_image"
+        identifiers = [
+          "output_resources.#.amis.#.image|@flatten|0",
+          "output_resources.#.amis.#.region|@flatten|0",
+        ]
+      }
+    }
+  }
+
+  resource "ec2.instances" {
+    ignore_attributes = ["launch_time"]
+
+    iac {
+      terraform {
+        type = "aws_instance"
+      }
+    }
+  }
+
+  resource "ec2.internet_gateways" {
+    filters = [
+      "NOT EXISTS (SELECT 1 FROM aws_ec2_internet_gateway_attachments a JOIN aws_ec2_vpcs v ON v.id=a.vpc_id WHERE a.internet_gateway_cq_id=c.cq_id AND v.is_default)",
+      "NOT EXISTS (SELECT 1 FROM aws_ec2_route_table_routes WHERE gateway_id=c.id AND destination_cidr_block='0.0.0.0/0')"
+    ]
+
+    iac {
+      terraform {
+        type = "aws_internet_gateway"
+      }
+    }
+  }
+
+  resource "ec2.nat_gateways" {
+    ignore_attributes = [ "arn", "create_time", "state", "vpc_id" ]
+    iac {
+      terraform {
+        type = "aws_nat_gateway"
+      }
+    }
+  }
+
+  resource "aws_ec2_nat_gateway_addresses" {
+    identifiers = [ "parent.id", "c.network_interface_id" ]
+    iac {
+      terraform {
+        type = "aws_nat_gateway"
+        identifiers = [ "id", "network_interface_id" ]
+      }
+    }
+  }
+
+  resource "ec2.network_acls" {
+    filters = [ "c.is_default!=true" ]
+
+    iac {
+      terraform {
+        type = "aws_network_acl"
+      }
+    }
+  }
+
+  #        resource "aws_ec2_network_acl_entries" {
```

Comment:
why is this commented out?

Reply:
Look one line down, it says:
`TODO: no CRC32 function, no data in tests to verify`